src/hooks/useQuizEvents/useQuizSkipClick.ts

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react';
+import { ActionAnswerQuestion, QuestionTypes } from '../../components/CioQuiz/actions';
+import { QuizAPIReducerState } from '../../components/CioQuiz/quizApiReducer';
+import { QuizLocalReducerState } from '../../components/CioQuiz/quizLocalReducer';
+import { QuizEventsReturn, OnQuizSkipQuestion } from '../../types';
+import { isFunction } from '../../utils';
+
+const useQuizNextClick = (
+  quizApiState: QuizAPIReducerState,
+  quizLocalState: QuizLocalReducerState,
+  dispatchLocalState: React.Dispatch<ActionAnswerQuestion>,
+  onQuizSkipQuestion?: OnQuizSkipQuestion
+): QuizEventsReturn.NextQuestion => {
+  const quizNexClickHandler = useCallback(() => {
```

Comment:
I believe you meant `quizSkipClickHandler`

Reply:
[39da1d4](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/132/commits/39da1d4e2a01f97440c535a87b5777047d6a8499)